api/docs/triager.dox

```diff
@@ -36,9 +36,11 @@
 
```

Comment:
> And remove reference to Jenkins since, while used, is done as part of a Github Action, and thus there's no separate place to look for failures.

nit: Generally the body should stand on its own, including repeating what is in the title if necessary: i.e., best to not start it with "And".  The title is not always right there depending on what history tool is used.